compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.td

```diff
@@ -142,6 +142,17 @@ def LLVMCPUTileAndFuse :
   ];
 }
 
+def LLVMCPUTileRootAndFuseProdCons :
```

Comment:
nit: I think it is okay to have a long name like `LLVMCPUTileRootAndFuseProducerConsumer` because it makes it more clear.